addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/data-sources/register/SubmitStep.js

```diff
@@ -160,9 +160,9 @@ class SubmitStep extends React.Component {
     const success = this.success;
     const failure = this.failure;
     const allFailed = this.allFailed;
-    const showNext = !allFailed && (failure || success);
+    const showNext = !allFailed && success;
```

Comment:
@SanketD92  per our discussion, you might want to undo this

Reply:
A quick recap of the discussion:

When any (but not all) of the steps during the registration have failed, there could be additional resources in the CloudFormation stack which still could be safely updated in the DS Account (for eg. a new study could be overlapping, but the parent bucket could be a newly registered valid resource). For this reason, we need to still show the Next button on the UI.

Reply:
The ultimate goal is to have a workable study which won't really happen until user takes the action of correcting whatever is it that went wrong. Updating the CloudFormation stack here doesn't really add much value since it needs to be corrected later anyways. I don't mind what we do either way but I thought it was important to call this bit out. @Sanket I will leave it on you to decide.

Reply:
The effort to compute all steps have either failed or been skipped is a bit too much just to disable the Next button. Rolling back changes to show the Next button and adding a warning message to alert the admin that whatever has failed would not make it to the CFN template if they wished to proceed anyway.
![Screen Shot 2021-01-11 at 1 17 43 PM](https://user-images.githubusercontent.com/20480854/104221781-658e4f80-540f-11eb-8cff-78b726d6ee2f.png)